generator/templates/base_struct_function.m.jinja2

```diff
@@ -34,31 +23,14 @@ NS_ASSUME_NONNULL_BEGIN
     {%- endfor %}
     return self;
 }
-{%- if deprecated or c.deprecated %}
-#pragma clang diagnostic pop
-{%- endif %}
 {% endfor -%}
 {% endblock -%}
 {%- block methods %}
 {%- for param in params %}
-{%- if deprecated or param.deprecated %}
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-{%- endif %}
 - (void)set{{param.origin|title}}:({{'nullable ' if not param.mandatory}}{{param.type_generic}}{{param.type_sdl|trim}}){{param.origin}} {
-{%- if deprecated or param.deprecated %}
-#pragma clang diagnostic pop
-{%- endif %}
     [self.{{parameters_store}} sdl_setObject:{{param.origin}} forName:SDLRPCParameterName{{param.method_suffix}}];
 }
-{% if deprecated or param.deprecated %}
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-{%- endif %}
 - ({{'nullable ' if not param.mandatory}}{{param.type_generic}}{{param.type_sdl|trim}}){{param.origin}} {
```

Comment:
```suggestion

- ({{'nullable ' if not param.mandatory}}{{param.type_generic}}{{param.type_sdl|trim}}){{param.origin}} {
```

Reply:
I added a newline between the setter and getter to keep line spacing consistent between each function consistent in the implementation file.